app/src/main/res/drawable/nft_background_card.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid
+        android:color = "@color/white"
+        />
+</shape>
```

Comment:
What I meant by the previous comment is that you should delete this file and just use android:background="@color/white" where this is being used instead of creating a drawable with just the white color.